app.js

```diff
@@ -15,6 +15,13 @@ const users = require('./routes/users');
 const responses = require('./routes/responses');
 const { requiresStaffRole } = require('./middlewares/auth');
 
+// TEST JESNKINS
+var JSON = function() {};
+
+var Promise = 22;
+
+let nesto = (  ) =>{{}}
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.extensionengine.com/coding_rules#rule_key=javascript%3AEmptyBlock)